src/login-page.less

```
@@ -1,8 +1,17 @@
-body.new-ui {
-  .login-page {
-    .overview {
+body.new-ui .login-page {
+  #kbc-landing-news {
+    padding-top: 45px;
+
+    @media (max-width: @screen-lg) {
+      padding-top: 0 !important;
+    }
```

@cJayyy Nemalo by to byt skor opacne? Akoze "mobile first" stylom?

```suggestion
    padding-top: 0 !important;

    @media (min-width: @screen-lg-min) {
      padding-top: 45px;
    }
```

---

Bylo by to ideální, ale tohle fixuje styly, který už tam jsou z toho indigo-ui; tam je právě někde `@media (max-width: @screen-lg)`.

---

Dobre